h5p-bildetema/src/components/Footer/Footer.module.scss

```diff
@@ -2,7 +2,6 @@
 
 .footer {
   background-color: $burgundy;
-  margin-top: $spacing--32;
```

Comment:
Burde flytte denne spacingen til body-elementet (eller tilsvarende) for hver side med margin-bottom. Slik at vi får litt mer mellomrom til footeren

Reply:
Brukte padding i stedet for margin på body element i topicRouteController

Reply:
Kan forresten se ut som margin ikke ble fjernet fra footer på mindre skjermstørrelser